composeApp/src/commonMain/kotlin/design-system/PrimaryMainButton.kt

```diff
@@ -0,0 +1,78 @@
+package design
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.RectangleShape
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+object PrimaryMainButtonConfig {
+   enum class ImageAlignment {
+       START, END
+   }
+}
+
+@Composable
+fun PrimaryMainButton(modifier: Modifier = Modifier,
+                      text: String,
+                      image: Painter? = null,
+                      imageAlignment: PrimaryMainButtonConfig.ImageAlignment = PrimaryMainButtonConfig.ImageAlignment.START,
+                      onClick: () -> Unit) {
+    @Composable
+    fun IconImage() {
+        if(image != null) Image(image,
+            modifier = Modifier.size(16.dp),
+            contentDescription = "Some image")
```

Comment:
Any string in the content description will be read by the TalkBack feature (accessibility). we probably don't want "some image" to be read here. Please pass this as a parameter or use null to suppress the talkback.

Reply:
Done